### ISSUE TITLE
Wrong URI in RESPMOD request

### ISSUE THREAD

**Issue by jhon-freitez-jemmic** [toolarium/toolarium-icap-client#5]
This is more a question that an issue.

Shouldn't the `RESPMOD` request use the exact port number in the URI in case of it is different to the default `1344`?

Currently, the code contains this:

```java
String requestBuffer = "RESPMOD icap://" + serviceInformation.getHostName() + "/" + serviceInformation.getServiceName() + " ICAP/" + requestInformation.getApiVersion() + NEWLINE 
...
```

https://github.com/toolarium/toolarium-icap-client/blob/master/src/main/java/com/github/toolarium/icap/client/impl/ICAPClientImpl.java#L285

Same situation occurs during the `OPTIONS` request:

https://github.com/toolarium/toolarium-icap-client/blob/master/src/main/java/com/github/toolarium/icap/client/impl/ICAPClientImpl.java#L91

**Comment by mepux** [comment]
Thanks, is fixed with new release

**Comment by mepux** [comment]
Resolved in release v1.2.0
